audit/Contribution_Service_Diagram_Attacks.drawio

```diff
@@ -0,0 +1 @@
CGCQEb4vT5jgofusaLlBp4M6FQXn0X9+jy/QpNOsw/gb5i2X2zaqZPvAKZpPTJ4mW0MPb3BC4moBLgjFNPq3uL1HAiZfRJbnuVcXZ/Y0RFNImF3xW7sCXa/zu3fJzcPPyv/f45uOHs/3N0V32xMhjBEgPMaFLvMAhDK4OoxcEb0IP8WUVdnSYc43xmg2qbPAronSXogk3FLOhJV0F6Vl079PP/PKpZaSHX3KnXt6nS8cHu+wgpGSXXGWbIBvg150pUwXsRw5Xx0eFy28R8VeIIpIOznFI07tUTXZcpmlK5ghviIuOENJMeROSBaJH5ql2MpFTOfcNKWSvEWa3R3ZsAkEBpP6PIhvClJsX+3kHwNmHFPMH4K+bQ+Kv5sBXpprREP889tllR5DPg6z3ArJujApkAwwq5AWQm2I8NQooP1a4+8EdWOPC3RqLcDfW7E8R9WzBkaCe3vYPGGzSbypxwQFjDsx26VN0t4YxMbbMjSviKdrIuR8ElzjAJF4LeAayPU7oiBL8DeXO2NoMmEet6g9EKLo/SrPsrJE6KjvheHtwyoCSji3zDpmpdERnTUJnM6ApxQoEN79vcHbiLIppec4mqOb6PiZPdp59WvDf5wsUr6BmK7IbTBZNzj8O0DbgMAU4zDIcqiaBozM0QPdoaGNFA2hjQ0PvHg0wVjRE2dD0odEwukdDz1ackQMQ48RHlJbh8XHqLTYKvXOezWBHIQ5RkShFl02011VxU13YJA/SjvvvpztbekNnq2lklQPTkGCZjTV2ydJvuMV+zO8ZLykCL9kCjyTPnV6VT86ICwn+DVu5uFBCmNJCMb/tH/t0FszSbb80D2oNeRCMigc1py0eBDULdc2DmSrvmgeLUabakAcPfPclz3Yt86DxJHmwpAedU3lQr1moax6URXXd82BtpkPKg8ezKt3rQX1cPKgWWWfPk4+1xaWFuuZBWSzbtS1uqgXVHnRgU19wZDpQiCv2/Phg/jNrFuqa/2TRewf8p57Cf0oP/NfUBmvj4j/Bfdvz44P5z6pZqGP+0wYtVxVDDeukmuQRXrb2dr+7WrTamIHHVbkA6pDA5wrQ9col58gB3SwosqlSG9WOBP6R7UUA9qDwnyT3arty/7iCZWPgR1anVssJ6ndrFN4hWGKIViuXENlzdyKpXJqujWbzdtLNmhDRGlk0MVi6WbUa+HfdC16LCq/x5qtx7cvRB6H7yXmwjpXXE91ukXnvw3urNZts2IGIXmU6qj4xPxJm0Ft3YeJLWWwLd7kJax7rRLmVhZDJFpS8aQl7cWvmsysERkzuoN24vrwz6BKvVpvQp/xZb9BqxqAUmTlawjX/uFkF5y7lFvKCGz/fZdwKZyi4xZFPfRyyKTNMKV7lJpwH/oKfoFiwzHhDAz9kJjfbtP0gD/gBxtc2irioZeMLerW9zqA+7qkhDhBDHBY6DBvj2A0VhDkqYwGG3Yn7gOrBqZWKASAemVeX3XdOza7n6ymHJcQBjKYUuUu2cjnZ0WZwg1TPQJYsuHFMC8C29KtQ9wOSbZmabFum3pWC1fRBJayQRKjJIRxzyLiK1Ysuma4MrHP3/VdPTCD3bH0QyCu6RARtVpXejrtj/omHCKgXxFnCPNez/QB0vy1ilnqXuDnpeJSiYNRL79zgf9J5ufHkR5qyiH/akWrdKXpNulWWakci1KpYqGrPbrawoVSv2ML4AUFvEjcQUgJdvha7Z+hBCtnHDfEfvo9RebT2ns/nmitNTXnmzDRawhnYRe2tZw0j+dSUTHtnDUXt49zCVtVqnL9vUMSjuBzAzxVa1dSK0GY6eThoB418HrKB8HimRDDK2sBpsyyV/9SMMignod/evrr7bZClbjYYmT3O3PcO9TS/hRVvzn++OlrYsGFIgqd+dbTeQhdVFaw3SQZVoYyg4fMFVfSpDKchqFn1u31QB+0CbqOsPrX6a/HXm9bQx5VY1MuO8zWLaxCZYUi8cnPWi9cYLxjSjCJrwiZGS4ToH5XWl8kuIgz+O4pJ/MKUWhtcYXJFoEQZtV0kl9GZbejG0W19zWXUcOyCjKqGWZZRWVlA7SxtpTfpAeu3Jv9IOWqavR3ZuzL0cvb24xKvIDddjCDVItJvfewoOM0lwTKtoiQ4ZUnotT5myHqAWvJA/mFB2NxHHMktDAJEn68XYor1aM2WaDijRy/E0LoDlkXXc0xWsXdJmOFBPJ8HXS510S+EsSVJ8fSLcYdZ2tsNc1E4wiHaZqHhMwb3TNwgAoaODY0OU7Of1h6kMbppyM+MvecTZvqeMcSa0GZmD518N2RvDmqxyBKVw5DIxQRJxp8v6rrQXCPzuHqGvezxdg37iseTXpYM+qXxB30q9vcX18btzw8O+XNmza2t/ubT2V/S91hVxDjzAN2nbX8XuQ5AN4BR5LuTI02AbHZKekdG4atz8yJOlh+2KgGteauprGIzkUWVtbGslEJAjmg/bX2q0NZsndrWLL4nQu+urbSaigU++xvRLSbf2GAAd3zrqQIpZf57lGiVslo4Y6Oeh5M0MvtnBiP06cPbiolcsShL/ytb0Q8X8klLStfsafgvPvvG/3hzRMk0fkFRSR9IGK/a9xMcA0srqwhDFpVbLUTlUujKjsF5DFSGwybiYleJmQlXnErhLOK/AhwXdzCJqYwDXg1Q1sT/wVxA9ukb4qR4kUTof8hRky5xfvs2uVxiT8YKtaEW8y+WpFrQL9QNeoOGsQYgbw2miuYMaREaNxyZg5oOoEzZ3/2PWuQ18RVTjfvDhU2fenf94VLiy/zVEbCooTujY9GhOM907GnW6pCFt4o2VcBpHGcJRdUO30wlpWW5OJNYD8XnhsdNmmhcGKf5hzYpprCrQLclAUZX/aZS6jV5p1J/AttUxIYTHWEDgVhzbCo04kJmz2pa8hqjKof/6Ths4rtRJG/a7ddfk/TO/xauSuEyxDcln+oDlRZq7T117PDwf8Ik0w//sw64+h8=</diagram></mxfile>
```

Comment:
Audit was merged into main, so I've merged main into staging to bring those latest changes